frontend/src/js/controllers/teamsCtrl.js

```diff
@@ -25,6 +25,7 @@
         vm.isPrev = '';
         vm.team.error = false;
         vm.showPagination = false;
+        vm.participantTeamId = null;
```

Comment:
Do we need this change? @Kajol-Kumari

Reply:
> Do we need this change? @Kajol-Kumari

@Sanji515 This line is actually not needed but i used it just to denote that currently participantTeamId does not have any value but it will have later on. But I will remove this line as it is not affecting any functionality.